zookeeper-server/src/main/java/org/apache/zookeeper/server/backup/BackupManager.java

```diff
@@ -606,9 +618,25 @@ public synchronized void initialize() throws IOException {
       } catch (Exception e) {
         throw new BackupException(e.getMessage(), e);
       }
+
+      // create metric-related classes for timetable backup
+      TimetableBackupStats timetableBackupStats = new TimetableBackupStats();
+      timetableBackupBean = new TimetableBackupBean(timetableBackupStats, serverId);
+      try {
+        MBeanRegistry.getInstance().register(timetableBackupBean, null);
+        LOG.info("BackupManager::initialize(): registered timetable backup bean {} with JMX.",
+            backupBean.getName());
+      } catch (JMException e) {
+        LOG.warn(
```

Comment:
NIT: Shall we use error? This seemed to be problematic if we blindly running service without metrics.